src/rdkafka_broker.c

```diff
@@ -5219,13 +5245,16 @@ int rd_kafka_brokers_add0(rd_kafka_t *rk, const char *brokerlist) {
         char *s_copy = rd_strdup(brokerlist);
         char *s      = s_copy;
         int cnt      = 0;
-        rd_kafka_broker_t *rkb;
-        int pre_cnt = rd_atomic32_get(&rk->rk_broker_cnt);
+        int pre_cnt  = rd_atomic32_get(&rk->rk_broker_cnt);
+        rd_sockaddr_inx_t *sinx;
+        rd_sockaddr_list_t *sockaddrList;
```

Comment:
lets not use camelCase here. Better to use same convention which is being used in librdkafka. `sockaddr_list`